src/Network/Network/PublicIpPrefix/NewAzurePublicIpPrefixCommand.cs

```diff
@@ -138,9 +138,18 @@ private PSPublicIpPrefix CreatePublicIpPrefix()
                 publicIpPrefix.PublicIpAddressVersion = this.IpAddressVersion;
             }
 
+            if (publicIpPrefix.PublicIpAddressVersion == MNM.IPVersion.IPv6 && (this.PrefixLength < 117 || this.PrefixLength > 127))
+            {
+                throw new ArgumentException($"Cannot validate argument on parameter 'PrefixLength' {this.PrefixLength}. Supply an argument that is greater than or equal to 117 and less than or equal to 127 and then try the command again");
```

Comment:
This text should be in a resource string

Reply:
These two strings are different with different prefix lengths for IPv4 and IPv6. Can you please advise if I am missing something ?

Reply:
updated PR by using resource string